### PR TITLE
docs_improvements_for_Waveforms&PulseShapes

### PR DESCRIPTION
**Pull request by GuillermoAbadLopez**
Improvement on `Waveforms` documentation for the selection process.

During which I did some improvement to the pulse_shape documentation as well.

### CODE REVIEW

File: src/qililab/pulse/pulse_shape/gaussian.py [hunk 1]
```diff
@@ -53,22 +64,23 @@ class Gaussian(PulseShape):
             :align: center
 
     Args:
-        num_sigmas (float): Sigma number of the gaussian pulse shape. Defines the height of the gaussian pulse.
+        num_sigmas (float): Sigma number of the gaussian pulse shape. Defines the width of the gaussian pulse.
```
Comment (jjmartinezQT) [inline src/qililab/pulse/pulse_shape/gaussian.py:67]: good catch

File: src/qililab/waveforms/iq_pair.py [hunk 2]
```diff
@@ -12,12 +12,15 @@
...
 
+"""IQPair dataclass."""
 from dataclasses import dataclass
 
 from qililab.waveforms.waveform import Waveform
 
 
 @dataclass
 class IQPair:  # pylint: disable=missing-class-docstring
+    """IQPair dataclass, containing the real (I) and imaginary (Q) parts of a signal."""
```
Comment (jjmartinezQT) [inline src/qililab/waveforms/iq_pair.py:23]: I am not really confortable with the 'imaginary' and 'real' part descriptions for I and Q after reading about it, I would rather stick to the 'in-phase' and 'quadrature' descriptions, respectively. What do you think?

**Review by jjmartinezQT — CHANGES_REQUESTED** [review]

File: src/qililab/waveforms/iq_pair.py (hunk 2 above)
Comment (GuillermoAbadLopez) [reply src/qililab/waveforms/iq_pair.py:23]: Sure yes! It's not really my domain, so I'll go with what you think its best!

Comment (GuillermoAbadLopez) [reply src/qililab/waveforms/iq_pair.py]: Changed!

**Review by jjmartinezQT — APPROVED** [review]

**Pull request merged**